### ISSUE TITLE
[Idea] Disable the covid videos YouTube recommends

### ISSUE THREAD

**Issue by SorcererSalamander** [YTVanced/Vanced#1111]
**Description**
YouTube keeps giving me this obnoxious shelf of covid-related videos. I'm sure I've closed it out at least a hundred times and it's not stopping. It's worse than ads. Can it be disabled?

_Attach images/videos if possible. Also add any links to possible other applications with the said feature already implemented._
![unknown-46](https://user-images.githubusercontent.com/87949274/126912043-2618e54f-45fa-46c3-871f-9c8ea1466c06.png)

**Comment by supernoodled** [comment]
+1 for this. I'm vaccinated and it's just annoying. I don't want these ads on my feed.

**Comment by milindgoel15** [comment]
Being vaccinated doesn't mean you won't get affected. YouTube is just trying to help you by telling you to be safe and do all preventive measures.

**Comment by SorcererSalamander** [comment]
> Being vaccinated doesn't mean you won't get affected. YouTube is just trying to help you by telling you to be safe and do all preventive measures.

I didn't wanna get into political degeneracy here but I guess it was inevitable. My take is that they're preemptively trying to avoid some internet news blowup about how "YouTube is promoting covid conspiracy theories". The videos aren't there for us, they're there for the media. Sure, a company may want to do their part to help stop the virus, but when they've given me a whole shelf about "how were black people involved of the creation of the COVID-19 vaccine?" about 10 times, something tells me they don't actually care about what I think.
And I'm not at all alone here. The videos that don't have likes hidden typically have a 1:10 like:dislike ratio at best, and when the comments aren't disabled/hidden, it's a cesspool of antivax conspiracy, which is notably disproportionate to how many people have gotten the vaccine. Go check for yourself next time you see it. 
I've probably just opened Pandora's box, whatever. Anyway, regardless of any good intention, regardless of right and wrong, the vast majority of users dislike these videos and would rather have them gone, and I see no reason why Vanced can't honor that. This isn't about covid, it's about user experience in a social media app.

**Comment by Harmonie666** [comment]
I've actually reported this issue several times, and unfortunately the devs marked it with a "wontfix". It saddens me that the devs won't put an end to this disturbing propaganda, that has destroyed many lives.
https://github.com/YTVanced/Vanced/issues/234

**Comment by SorcererSalamander** [comment]
> I've actually reported this issue several times, and unfortunately the devs marked it with a "wontfix". It saddens me that the devs won't put an end to this disturbing propaganda, that has destroyed many lives.
> Here you go:
> #234

Ok that's a little dramatic

**Comment by umbrellacorp53** [comment]
https://github.com/YTVanced/Vanced/issues/234#issuecomment-1013595001

**Comment by DraconicNEO** [comment]
> [#234 (comment)](https://github.com/YTVanced/Vanced/issues/234#issuecomment-1013595001) I'm trying to figure out if I should delete the original comment and move it here or what. I'm going to leave the link above and add a quote reply here.
> 
> > I see that this is marked as "won't fix"... Look, I know you don't want to be "on the same side" as the wacky anti-vaxxers. That's not what is being requested of you, and not what this toggle would represent.
> > I have family who are dead and in ICU and medically induced coma from covid. I have been vaccinated and I am educated on the current state of things. So is everyone I know and I try to convince my folks to get vaccinated.
> > I get on YouTube to choose what content I see. I installed vanced to allow me to choose who I support and by what means. I would like the choice to turn off invasive news stories that seem to pop up when I'm not prepared. I do not want to think about my cold dead uncle, the funeral, my elderly parents who refuse the vaccine, my only uncle who is left but in ICU, the children in hospice that I passed on my way to see him, or the terrifying supply chain issues, food shortages, the economic issues, new variants, etc. when I get on this app.
> > Sure, it should be left alone by default so your work does not have a negative impact, but for those of us who are trying to learn and stay focused, need to escape, or are dealing with grief, anxiety, depression, etc. we should not be FORCED to see these reminders of the horrors we're living through, especially at random, without consent, and with no warning.
> > The only people who would choose to turn this off are either looking for solice from the reminders of tragedy that are seemingly inescapable and non-negotiable, or are the types who are entrenched into a political ideology and are unwilling to listen and therefore the message is lost on them anyway... So choosing to ignore this issue just to spite people who will not change is only hurting the former who frankly could use a brief escape. Either way, not giving the choice, when it is clearly within your power is not helpful.

Agreed, people don't want to hide this because they're Crazy Conspiracists, they want to hide them because they don't want to be constantly reminded of the pain they went through and will likely keep going though.
